Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteScrollingCoordinator.mm

```diff
@@ -59,17 +59,21 @@
 RemoteScrollingCoordinator::RemoteScrollingCoordinator(WebPage* page)
```

Comment:
nit: Definitely doesn't have to be in this patch but this should take in a reference since we expect `page` to be non-null.